int32 ID
string roomName
float32 batteryStatus